patroni/ctl.py

```diff
@@ -515,7 +523,14 @@ def reload(obj, cluster_name, member_names, force, role):
 def restart(obj, cluster_name, member_names, force, role, p_any, scheduled, version, pending, timeout):
     cluster = get_dcs(obj, cluster_name).get_cluster()
 
-    members = get_members(cluster, cluster_name, member_names, role, force, 'restart')
+    if scheduled is None and not force:
+        next_hour = (datetime.datetime.now() + datetime.timedelta(hours=1)).replace(second=0, microsecond=0).isoformat()
```

Comment:
I specifically [mentioned](https://github.com/zalando/patroni/pull/1119#discussion_r308251786) the usage of `strftime('%Y-%m-%dT%H:%M')` in order to avoid having ":00" seconds printed.

Reply:
Please also do the same with https://github.com/zalando/patroni/pull/1119/files#diff-b558c390fbbb65333f66b7b6a8fe73cdL638